app/konnect/dev-portal/applications/dynamic-client-registration/curity.md

```diff
@@ -0,0 +1,118 @@
+
+To get started configuring Curity, log in to your Curity dashboard and complete the following:
+
+1. Select the **Profiles** tab near the top of the dashboard.
+
+2. Select an existing **Token Service Profile** in the **Profiles** diagram, or create a new one if necessary.
+
+3. Complete the following sections using the **Token Service Profile** you have selected.
+
+### Configure token issuer
+
+1. Select **Token Service > Token Issuers** from the menu.
+
+2. Enable the setting **Use Access Token as JWT**.
+
+3. Add a new token issuer using **New Token Issuer** button.
+
+4. Fill in the following values for the token issuer, and click **create**:
+    * Name: `userinfo`
+    * Issuer Type: `jwt`
+    * Purpose Type: `userinfo`
+
+5. In the displayed "Edit Custom Token Issuer" form, select the desired values for **Tokens Data Source ID**, **Signing Key**, and **Verification KeyStore**.
+
+### Create a client
+
+1. Select **Token Service > Clients** from the menu.
+
+2. Click **New Client** to add a new client.
+
+3. Give the client a unique and descriptive name, noting it for later use.
+
+4. Click the **Capabilities** in the overview diagram to add a capability to the **client**.
+
+5. Select **Client Credentials** and click **next**.
+
+6. Set the **Authentication Method** to `secret` and generate a secret, copy it for later use, and click **next**.
+
+   {:.important}
+   > **Important:** Store the secret in a place you can reference, because it will not be visible after this step.
+
+### Enable Dynamic Client Registration
+
+1. Select **Token Service > General > Dynamic Registration** from the menu.
+
+2. Click **Enable Dynamic Client Registration**.
+
+3. Ensure **Non Templatized** and **Dynamic Client Management** are both enabled and click **next**.
+
+4. Select the desired client data source and click **next**.
+
+5. Select the **Authentication Method** `authenticate-client-by`, add the name of the Client that was created above, and click **next**.
+
+6. Select the desired nodes to enable DCR on and click **next**.
+
+## Configure the Dev Portal
+
+Once you have Curity configured, you can set up the Dev Portal to use Curity for dynamic client registration (DCR).
+
+1. Sign in to {{site.konnect_short_name}}, then select {% konnect_icon dev-portal %} **Dev Portal** from the menu.
+
+2. Click **Settings** to open the Dev Portal settings.
+
+3. Click the **Application Setup** tab to open the DCR settings for your Dev Portal.
+
+4. Select **Curity** as the external identity provider.
+
+5. Enter the **Issuer URL** for your authorization server, it will look something like `https://<YOUR_CURITY_INSTANCE_DOMAIN>/oauth/v2/oauth-anonymous/.well-known/openid-configuration`
```

Comment:
```suggestion
5. Enter the **Issuer URL** for your authorization server, it will look something like `https://CURITY_INSTANCE_DOMAIN/oauth/v2/oauth-anonymous/.well-known/openid-configuration`
```